css/properties/border-block-end-color.json

```diff
@@ -0,0 +1,50 @@
+{
+  "css": {
+    "properties": {
+      "border-block-end-color": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-block-end-color",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "webview_android": {
```

Comment:
I think we want "webview_android", "chrome" and "chrome_android" in the browser list, in that order. Chrome_android is missing here (in all cases) and should be false like "chrome". That's what I had for HTML.

Reply:
Should I be adding `chrome_android` (or any other browser) in cases where the wiki page's table didn't have any data? Up to now I had been omitting browsers which were omitted in the original table.